.github/scripts/upload-rpm-to-cloudsmith.sh

```diff
@@ -8,7 +8,7 @@ if (( $# < 1 )); then
   exit 1
 fi
 
-if [[ -z $CLOUDSMITH_API_TOKEN ]]; then
+if [[ -z "${CLOUDSMITH_API_TOKEN:-}" ]]; then
```

Comment:
I don't understand the rationale for this change.  The `:-` part does literally nothing; the braces are nice for readability, but not needed; I would prefer to not use double brackets (they are bash only) but if you use them the quotes aren't needed either.  Consider `if [ -z "${CLOUDSMITH_API_TOKEN}" ];` or even
`if [ "${CLOUDSMITH_API_TOKEN} = "" ];`

Reply:
Wouldn't  `if [ "${CLOUDSMITH_API_TOKEN} = "" ];` fail if the variable is not defined because of the `set -u` ?